java-client/src/main/java/org/apache/pegasus/security/KerberosProtocol.java

```diff
@@ -161,19 +173,16 @@ private static Configuration getLoginContextConfiguration(String keyTab, String
       @Override
       public AppConfigurationEntry[] getAppConfigurationEntry(String name) {
         Map<String, String> options = new HashMap<>();
-        // TGT is obtained from the ticket cache.
-        options.put("useTicketCache", "true");
```

Comment:
This is for keytab based login so we should remove useTicketCache, but do we support login use ticket cache? There is another method for ticker cache based login?

Reply:
yes, we can use ticket cache without keytab.  these can be done in the same method like hadoop